src/GovUk.Education.ExploreEducationStatistics.Admin/Services/Interfaces/IEmailTemplateService.cs

```diff
@@ -23,5 +22,7 @@ public interface IEmailTemplateService
             string email,
             Release release,
             ReleaseRole role);
+
+        public Either<ActionResult, Unit> SendReleaseApproverEmail(string email, Release release);
```

Comment:
As it's an interface, every method declared in it is public by default, so no need to include the `public` keyword here :slightly_smiling_face: